### ISSUE TITLE
How to save image, mask and prediction to jpeg

### ISSUE THREAD

**Issue by vithmel** [ZFTurbo/ZF_UNET_224_Pretrained_Model#12]
Hi I am newbie in DL.
I have some questions(Keras + Tensorflow):
For example I generate image and mask(train_infinite_generator.py) `image, mask = gen_random_image()`
1. How I can save image and mask to jpeg? I tried to use Pillow but I recieve something wrong
 
2. How to do predictions. I do:
```
image, mask = gen_random_image()
prediction = model.predict(np.expand_dims(image, axis=0))
```
As I wrote above I can not to see result and I do not know is everything that I do right.
 
3. How I can save prediction to jpeg(prediction shape is (1, 224, 224, 1)), I tried:

```
from PIL import Image
i3 = prediction[0, :, :]
i3 = np.rollaxis(i3, 2, 0)
img2 = Image.fromarray(i3[0], mode='L')
img2.convert(mode='RGB')
img2.save('my.jpg')
```

	
Result:
![my](https://user-images.githubusercontent.com/15888026/47787648-3270b180-dd18-11e8-936e-b334a2382b60.jpg)

**Comment by ZFTurbo** [comment]
I usually do it like that:

```
import cv2
i3 =prediction[0, :, :, 0] # Shape is (224, 224) here
i3 = (i3*255).astype(np.uint8)
cv2.imwrite('my.jpg', i3)
```
